src/reducers/toolbarOverlay.js

```diff
@@ -0,0 +1,16 @@
+export const showOverlay = (state, { id, name, description }) => ({
+    toolbarOverlay: {
+        showOverlay: true,
+        shortcutId: id,
+        shortcutName: name,
+        shortcutDescription: description
+    }
```

Comment:
It seems like all of this data could be derived from the id.

e.g. `toolbarOverlay: id`

Reply:
This question still needs to be addressed. Why have all of the other data duplicated when they can be derived from the id?